src/psyclone/tests/dynamo0p3_quadrature_test.py

```diff
@@ -45,7 +45,7 @@
 from fparser import api as fpapi
```

Comment:
pylint: not strictly yours but if you don't mind ...

C: 45, 0: third party import "from fparser import api as fpapi" should be placed before "from dynamo0p3_build import Dynamo0p3Build" (wrong-import-order)

Reply:
Done.